### PR TITLE
Update the main README.md doc

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Updated the main `README.md` document for the Generic Documentation component wth more details and link to the `documentation-component` repository.

**Related issue(s)**
See also #1029

### CODE REVIEW

File: components/generic-documentation/README.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
-# Documentation Component
+# Generic Documentation
 
 ## Overview
 
-Documentation Component is a generic, reusable React component that allows you to render:
+Generic Documentation is a Console-dedicated wrapper for the [Documentation Component](https://github.com/kyma-incubator/documentation-component/blob/master/README.md) - a generic, reusable React set of libraries allowing you to render specifications of these types:
```
Comment (mmitoraj) [inline components/generic-documentation/README.md]: is it `reusable React` or `reusable, React`?
```suggestion
Generic Documentation is a Console-dedicated wrapper for the [Documentation Component](https://github.com/kyma-incubator/documentation-component/blob/master/README.md) - a generic, reusable, React set of libraries allowing you to render specifications of these types:
```

File: components/generic-documentation/README.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+
 - Markdown
 - OpenAPI
 - AsyncAPI
-- OData
\ No newline at end of file
+- OData
+
+Generic Documentation provides individual layouts for documents and specifications rendered under the **docs-ui**, **Service Catalog**, and **Compass** views in the Console. These layouts are compliant with [Fiori 3 Fundamentals](https://sap.github.io/fundamental/) styling, for example in terms of navigation and tabs. It also downloads specification sources to the Console from such storage systems as MinIO.
```
Comment (mmitoraj) [inline components/generic-documentation/README.md]:
```suggestion
Generic Documentation provides individual layouts for documents and specifications rendered in the **docs-ui**, **Service Catalog**, and **Compass** views in the Console. These layouts are compliant with [Fiori 3 Fundamentals](https://sap.github.io/fundamental/) styling, for example in terms of navigation and tabs. Generic Documentation also downloads specification sources to the Console from such storage systems as MinIO.
```

File: components/generic-documentation/README.md (hunk 1 above)
Comment (kazydek) [reply components/generic-documentation/README.md]: I think I would go without the comma as the latter part is only a sentence fragment without a subject

File: components/generic-documentation/README.md (hunk 2 above)
Comment (kazydek) [reply components/generic-documentation/README.md]: I would keep "under"

**Pull request merged**